### PR TITLE
fix comments and setup ts

### PR DESCRIPTION
**Pull request by Alexandr98**

### CODE REVIEW

File: src/function/pow.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function pow(x, n) {
+  let res = x;
+  if (Numver.isNan(x)) {
+   return ('Not a number');
+  } for (let i = 1; i < n; i++) {
+      res = res * x;
```
Comment (LimarenkoDenis) [inline src/function/pow.js]: prettify code

File: src/loops/evenNumbers.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export function evenNum(i, a) {
+  let arr = [];
+  for (i = 0; i < a; i++) {
+    if (i % 2 !== 0) continue;
+      arr.push(i);
```
Comment (LimarenkoDenis) [inline src/loops/evenNumbers.js]: inverse logic
use `{}` for if construction, not in one line, setup eslint rule for that

File: src/recursion/sumTo.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export function sumTo(n) {
+  if (n == 1) return 1;
```
Comment (LimarenkoDenis) [inline src/recursion/sumTo.js]: use {} for if construction, not in one line,
`===` strict comparison

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
```
Comment (LimarenkoDenis) [inline .eslintrc.js]: if you use tslint then delete eslint

File: src/recursion/sumTo.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+  return n + sumTo(n - 1);
+}
+
+alert( sumTo(100) );
```
Comment (LimarenkoDenis) [inline src/recursion/sumTo.js]: don't use alert

File: .eslintrc.js (hunk 4 above)
Comment (NepipenkoIgor) [reply .eslintrc.js]: ++

File: src/function/min.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+export function min(a, b) {
+  if (a < b) {
+    return a;
+  }
```
Comment (NepipenkoIgor) [inline src/function/min.js]: formating

File: src/function/pow.js (hunk 1 above)
Comment (NepipenkoIgor) [reply src/function/pow.js]: ++

File: tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    // "noUnusedLocals": true,                /* Report errors on unused locals. */
+    // "noUnusedParameters": true,            /* Report errors on unused parameters. */
+    // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
+    // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
+
+    /* Module Resolution Options */
+    // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
+    // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
+    // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
```
Comment (NepipenkoIgor) [inline tsconfig.json]: if you use TS then remove js ( rewrite)

**Pull request closed without merging**